torch/csrc/jit/api/compilation_unit.h

```diff
@@ -117,7 +118,7 @@ struct TORCH_API CompilationUnit {
     if (shouldMangle) {
       name = mangle(name);
     }
-    auto fn = torch::make_unique<Function>(
+    auto fn = torch::make_unique<FunctionImpl>(
```

Comment:
FunctionImpl -> GraphFunction to be more specific since BuiltinFunction is also an implemenation.

Reply:
We already have a concept called builtin functions btw: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/frontend/builtin_functions.cpp

Should we rename something?